fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -23,13 +24,57 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
     """
 
     # Constants
-    separate = lambda: print("-" * 10)
     fides_dir_name = ".fides"
     fides_dir_path = f"{fides_directory_location}/{fides_dir_name}"
     config_file_name = "fidesctl.toml"
     config_path = f"{fides_dir_path}/{config_file_name}"
     config = ctx.obj["CONFIG"]
 
+    with_analytics(
+        ctx,
+        init_fides,
+        fides_dir_path=fides_dir_path,
+        config_path=config_path,
+        config=config,
+    )
```

Comment:
@PSalant726 I put this together last night as an option for discussion today (the diff looks a bit wonky unfortunately). It should effectively align it with other commands implementation to use the `with_analytics()` function and attempt to send an event for the `init` command if opted in. 

Unfortunately, the client creation is still an issue here, so wanted to get your thoughts. I feel like there are three options here:

1. Continue to modify this command to be able to send events in line with other commands (resolve client creation issues)
2. Revert this change and continue to treat the `init` command special, having a separate client/event send operation if opted in
3. Revert this change and determine to not send events for the `init` command

Let me know what you think, lots of opportunities to change this around and improve organization so open to other ideas - thanks!